### DISCUSSION TITLE
Is there a way to hide omitted matches entirely?

### DISCUSSION THREAD

**Discussion by Rogach** [BurntSushi/ripgrep#1779] [Q&A] [answered]
For example, when I use `-M` flag to hide matches on long lines I still get lots of results like the following:

```
some-file.svg
86:[Omitted long line with 3 matches]
91:[Omitted long line with 2 matches]
```

And there can be quite a lot of such matches, to the point when it is hard to find actually relevant non-omitted ones.

Is there some flag that hides such matches completely? I wasn't able to find anything like that in the man pages.

**Comment by BurntSushi** [comment, marked as answer]
No. The simplest way would be to filter the output. For example, with `rg -vF '[Omitted long line with'`.

**Reply by Rogach** [reply]
While that works, the pretty formatting gets lost (since `rg` output is piped). Would a PR with a flag to hide omissions be acceptable?

P.S. Thank you for all your work!

**Reply by BurntSushi** [reply]
I doubt it. I'm not convinced it's worth it. Sorry.
